app/controllers/keywords_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class KeywordsController < ApplicationController
+  def create
+    form = CsvUploadForm.new
+
+    if form.save create_params[:file]
+      flash[:notice] = I18n.t 'csv.upload_success'
+    else
+      flash[:alert] = form.errors.full_messages
+    end
+
+    redirect_to keywords_path
```

Comment:
Should we define the `keywords#index` action now? otherwise I'm not sure where does the `keywords_path` come from 🤔

Reply:
Fixed in 101795de4440249c337c9f901659c0c01accd3c4